apps/website/netlify/edge-functions/strangler.ts

```diff
@@ -0,0 +1,85 @@
+import { Config, Context } from 'https://edge.netlify.com';
+
+type LegacyHost = {
+  url: string;
+  urlFilter?: (url: URL) => boolean;
+  responseFilter?: (response: Response) => boolean;
```

Comment:
How do you think, is there a chance that we will want to modify the `response`? If yess, maybe it should return `Response | null`?